onnx/defs/tensor/utils.cc

```diff
@@ -236,6 +240,14 @@ void resizeShapeInference(InferenceContext& ctx) {
   } // nullptr != scales
 }
 
+void resizeShapeInference_opset13_to_18(InferenceContext& ctx) {
+  resizeShapeInferenceVersioned(ctx, 18);
+}
+
+void resizeShapeInference_opset11_to_12(InferenceContext& ctx) {
+  resizeShapeInferenceVersioned(ctx, 12);
```

Comment:
nit: I think it is better to pass in the value 11, because the op is defined in opset version 11.